mansplainer.py

```diff
@@ -52,9 +52,12 @@ def incoming_sms():
             model = appC.models.get("general-v1.3")
             imageresult = model.predict_by_url(url=image_URL)
 
-            mansplanation = None
-            for tag in imageresult["outputs"][0]["data"]["concepts"]:
-                mansplanation = wiki.getSummaryFromWiki(tag["name"])
+            resultList = imageresult["outputs"][0]["data"]["concepts"]
+            selectedConcept = random.choice(resultList)
+            mansplanation = wiki.getSummaryFromWiki(selectedConcept)
+            while (mansplanation == None) : #search again if previous search did not find anything 
```

Comment:
Also, don't you think this may lead to infinite loop in case there's no article for all tags (even though it's small chance)